tap_shopify/streams/base.py

```diff
@@ -91,6 +113,13 @@ class Stream():
     def __init__(self):
         self.results_per_page = Context.get_results_per_page(RESULTS_PER_PAGE)
 
+        self.request_timeout = REQUEST_TIMEOUT # set default timeout
+        timeout_from_config = Context.config.get('request_timeout')
+        # updated the timeout value if timeout is passed in config and not from 0, "0", ""
+        if timeout_from_config and float(timeout_from_config):
```

Comment:
Why do we need to write this code 2 times, can't we put this inside one function and use it?

Reply:
Added a function that returns the timeout value.